modules/methods.js

```diff
@@ -0,0 +1,28 @@
+import resetIndex from './resetIndex.js';
+
+export default class {
+  constructor(list) {
+    this.list = list;
+  }
+
+    addItem = (value) => {
+      const ind = this.list.length;
+      if (value) {
+        value = value.trim();
+        const listItem = {
+          description: value,
+          completed: false,
+          index: ind,
+        };
+        this.list.push(listItem);
+        localStorage.setItem('listData', JSON.stringify(this.list));
+      }
+    }
```

Comment:
- [ ] The requirement for the to-do list item indexes is that they should be equal to the available items. If we have 5 items, then the last item index should be 5.

![edit](https://user-images.githubusercontent.com/53124548/213645182-9492461e-c04f-436b-84b4-efee058949fc.png)